sub-packages/bionemo-geneformer/src/bionemo/geneformer/scripts/train_geneformer.py

```diff
@@ -72,10 +73,10 @@ def main(
     wandb_entity: Optional[str] = None,
     wandb_project: Optional[str] = None,
     wandb_offline: bool = False,
-    wandb_tags: Optional[List[str]] = None,
+    wandb_tags: List[str] = [],
```

Comment:
Please revert this change. Mutable default values are global in Python: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
The right pattern to use here for this is to check and initialize to a fresh empty list each time the function is called:
```
if wandb_tags is None:
  wandb_tags = []
```

Reply:
Sure, I can do that.